packages/components/src/FormatFile/FormatFile.tsx

```diff
@@ -1,91 +1,129 @@
-import React, { useState } from "react";
-import filesize from "filesize";
-import { IconNames } from "@jobber/design";
+import React from "react";
+import classnames from "classnames";
+import getHumanReadableFileSize from "filesize";
 import styles from "./FormatFile.css";
-import { Button } from "../Button";
-import { Icon } from "../Icon";
-import { Typography } from "../Typography";
-import { ProgressBar } from "../ProgressBar";
+import { FormatFileDeleteButton } from "./FormatFileDeleteButton";
+import { InternalThumbnail } from "./InternalThumbnail";
 import { FileUpload } from "../InputFile";
+import { Text } from "../Text";
+import { ProgressBar } from "../ProgressBar";
 
 interface FormatFileProps {
   /**
    * File upload details object. (See FileUpload type.)
    */
   readonly file: FileUpload;
 
+  /**
+   * To display as either a file row or thumbnail
+   *
+   * @default "expanded"
+   */
+  readonly display?: "expanded" | "compact";
+
+  /**
+   * The base dimensions of the thumbnail
+   *
+   * @default "default"
+   */
+  readonly displaySize?: "base" | "large";
```

Comment:
based on the values, it's base now
```suggestion
   * @default "base"
   */
  readonly displaySize?: "base" | "large";
```